### ISSUE TITLE
Add warm-up and slow-down feature in video training

### ISSUE THREAD

**Issue by vlcvboyer** [GoldenCheetah/GoldenCheetah#1689]
Some video start directly with climbing...
It will be useful to propose configurable warm up just before starting video with slope simulation mode
Slow down will also be appreciated

**Comment by adutton** [comment]
There is another feature request for this functionality that has a bit more discussion: https://github.com/GoldenCheetah/GoldenCheetah/issues/3632 

This could probably be closed as a duplicate of that.

**Comment by amtriathlon** [comment]
Ok, let's keep the other one, this one never got traction anyway.
